### ISSUE TITLE
Support for SSD1326

### ISSUE THREAD

**Issue by iggymayer** [rm-hull/luma.oled#254]

**Comment by rm-hull** [comment]
No, SSD1326 is not presently supported, only because we do not have a device to test against. If you (or anyone else) wants to fund or donate/loan a display to add support for this device please get in touch.

**Comment by thijstriemstra** [comment]
why close this?
